src/main/java/com/chessmates/controller/EntityController.groovy

```diff
@@ -22,12 +20,12 @@ class EntityController {
      * Return a list of all team players
      */
     @GetMapping(value = 'players')
-    List<Player> getPlayers() { entityService.getPlayers() }
+    List getPlayers() { entityService.getPlayers() }
 
     /**
      * Return a list of all unique games played between all team members.
      */
     @GetMapping(value = 'games')
-    List<Game> getGames() { entityService.getGames() }
+    List getGames() { entityService.getGames() }
```

Comment:
As discussed it's not really JSON - I'll change the LichessApi to be more explicit about returning a page.